docs/guides/security-guide.md

```diff
@@ -132,18 +119,23 @@ by the ca.crt (can copy the generated files from the server if client is not
 on the same system as server).
 
 ### P4RT client
-
-The p4rt-ctl (P4RT client) will default to communicate via secure mode
-(port 9559). If certificates are not available, the P4RT client will attempt a
+The P4Runtime Control client will default to communicate via secure mode using
```

Comment:
```suggestion
The P4Runtime Control client will default to communicating in secure mode using
```
A verb as a noun is a _gerund_, which calls for an _-ing_ ending.

"via secure mode" doesn't sound quite right to me. _via_ means 'through'. I think secure mode is something you're _in_, rather than something you _go through_.